core/tests/test_utils.py

```diff
@@ -2555,6 +2555,23 @@ def set_topic_managers(
                     'topic_id': topic_id
                 }, csrf_token=self.get_new_csrf_token())
 
+    def set_translation_coordinators(
+        self, translation_coordinator_usernames: List[str], language_id: str
+    ) -> None:
+        """Sets role of given users as TRANSLATION_COORDINATOR.
+
+        Args:
+            translation_coordinator_usernames: list(str). List of usernames.
+            language_id: str. The language Id.
+        """
+        with self.super_admin_context():
+            for username in translation_coordinator_usernames:
+                self.put_json('/translationcoordinatorrolehandler', {
+                    'username': username,
+                    'action': 'assign',
+                    'language_id': language_id
+                }, csrf_token=self.get_new_csrf_token())
```

Comment:
This method needs to be covered by tests. From what I've read, the features we want to provide by adding this method includes:
- Our testing utilities `gives_translation_coordinator_role_in_a_language_to_users`
  - **Motivation of scenario**: Users that are not translation coordinators will become translation coordinators 
  - **Action**: Giving translation coordinator role in a language to users without the role
  - **Expected result**: All mentioned users are translation coordinators in the given language 
- Our testing utilities `adds_another_translation_coordinator_language_to_translation_coordinator`
  - **Motivation of scenario**: Translation coordinator(s) can be assigned new languages without replacing their existing ones
  - **Action**: Giving a unique translation coordinator language multiple times
  - **Expected result**: The user(s) should have all of the given translation coordinator languages

Reply:
Yeah, you are correct. Sorry I missed out on writing test for this code. Thanks for looking out.